lib/cli.js

```diff
@@ -115,28 +137,29 @@ Promise.resolve()
         if (options.updateAll) {
             return updateAll(currentState);
         }
+
         if (options.update) {
             return interactiveUpdate(currentState);
         }
 
         return staticOutput(currentState);
     })
-    .catch(err => {
-        console.log(err.message);
+    .catch(error => {
+        console.log(error.message);
+
         if (options.debug) {
-            console.log(createCallsiteRecord(err).renderSync());
+            console.log(createCallsiteRecord(error).renderSync());
         } else {
             console.log('For more detail, add `--debug` to the command');
         }
+
         process.exit(1);
     });
 
-const SUPPORTED_INSTALLERS = ['npm', 'pnpm', 'ied'];
+// eslint-disable-next-line unicorn/prefer-set-has
+const SUPPORTED_INSTALLERS = ['npm', 'pnpm', 'ied', 'yarn'];
```

Comment:
I think that `Set` would be nice here?

```suggestion
const SUPPORTED_INSTALLERS = new Set(['npm', 'pnpm', 'ied', 'yarn']);
```

```diff
- SUPPORTED_INSTALLERS.includes(preferredPM.name)
+ SUPPORTED_INSTALLERS.has(preferredPM.name)
```